### PR TITLE
update the seed

### PR DESCRIPTION
**Pull request by Thibault-Sordet**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -8,13 +8,61 @@
...
+
+10.times do
+  users = User.new(
+    email: Faker::Internet.email,
+    password: "azerty",
+    username: Faker::Internet.username,
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name
+  )
+  users.save!
```
Comment (NicolasNeiman) [inline db/seeds.rb:22]: I would recommand using save rather than save!

File: db/seeds.rb [hunk 2]
```diff
@@ -8,13 +8,61 @@
...
+end
+
+20.times do
+  bookings = Booking.new(
+    start_date: Faker::Date.backward(days: rand(1..25)),
+    end_date: Faker::Date.forward(days: rand(1..25)),
+    user_id: u_ids.sample(1).first.to_i,
+    star_id: s_ids.sample(1).first.to_i
+  )
+  bookings.save!
```
Comment (NicolasNeiman) [inline db/seeds.rb:59]: I would recommand using save rather than save!

File: db/seeds.rb [hunk 3]
```diff
@@ -8,13 +8,61 @@
...
   star = Star.new(
     name: Faker::Space.unique.star,
-    price: rand(1..20000),
-    user_id: rand(2..6)
+    price: rand(100..10000),
+    user_id: u_ids.sample(1).first.to_i,
+    galaxy: Faker::Space.galaxy,
+    description: Faker::Lorem.paragraph
   )
   star.save!
```
Comment (NicolasNeiman) [inline db/seeds.rb:42]: I would recommand using save rather than save!

File: db/seeds.rb (hunk 1 above)
Comment (Joannacha) [reply db/seeds.rb:22]: why ?

**Comment by Joannacha** [comment]
Très bien la nouvelle description
seed testée en local
ajout de delete_all pour Booking & Stars

**Review by Joannacha — APPROVED** [review]
Perfect for me

**Pull request merged**